include/mbedtls/mbedtls_config.h

```diff
@@ -1769,8 +1769,19 @@
  * This setting allows support for cryptographic mechanisms through the PSA
  * API to be configured separately from support through the mbedtls API.
  *
- * Uncomment this to enable use of PSA Crypto configuration settings which
- * can be found in include/psa/crypto_config.h.
+ * When this option is disabled, the PSA API exposes the cryptographic
+ * mechanisms that can be implemented on top of the `mbedtls_xxx` API
+ * configured with `MBEDTLS_XXX` symbols.
```

Comment:
Are these not macros, not symbols?

Reply:
I think in C terms “symbol” is incorrect, but we have a long habit of using “symbol” and not “macro” in relation with configuration options, so this is a preexisting issue which is out of scope here.